README.md

```diff
@@ -1,13 +0,0 @@
-# Google Student Training in Engineering Program
```

Comment:
You should update the readme instead of deleting it.

https://github.com/bhassan6621/StepTraining/blob/master/walkthroughs/week-1-setup/github-setup-walkthrough.md#modify-readme

This should also be in a separate pull request from updating styling of your portfolio.

Reply:
Doesn't look resolved?